packages/expo-device/src/ExpoDevice.web.ts

```diff
@@ -25,7 +25,15 @@ export default {
   get deviceYearClass(): null {
     return null;
   },
-  get totalMemory(): null {
+  get totalMemory(): number | null {
+    if ('deviceMemory' in navigator) {
+      // @ts-ignore
+      const { deviceMemory = 0 } = navigator;
```

Comment:
If this value ends up being zero, we probably should return `null` to have one consistent way of saying "we don't know."